client/lib/abtest/active-tests.js

```diff
@@ -103,4 +103,12 @@ export default {
 		allowExistingUsers: true,
 		localeTargets: 'any',
 	},
+	calypsoifyGutenberg: {
+		datestamp: '20181112',
```

Comment:
Since we want to deploy this tomorrow, I think we should update this to `20181113`